pyproject.toml

```diff
@@ -26,9 +26,9 @@ ignore = [
 # Disabled for now: https://github.com/PyCQA/pylint/issues/3584
 jobs = 2
 # Disabled pylint_strict_informational for codefactor
-load-plugins = [
-    "pylint_strict_informational",
-]
+#load-plugins = [
+#    "pylint_strict_informational",
+#]
```

Comment:
Why do we want to lower the restriction?

Reply:
That was a quick fix for this:
```
... pylint-ing ...
************* Module Command line or configuration file
Command line or configuration file:1:0: E0013: Plugin 'pylint_strict_informational' is impossible to load, is it installed ? ('No module named 'pylint_strict_informational'') (bad-plugin-value)
```

Reply:
Also, the Core setttings don't have this, but several others:
```
[tool.pylint.MAIN]
py-version = "3.9"
ignore = [
    "tests",
]
# Use a conservative default here; 2 should speed up most setups and not hurt
# any too bad. Override on command line as appropriate.
jobs = 2
init-hook = """\
    from pathlib import Path; \
    import sys; \
    from pylint.config import find_default_config_files; \
    sys.path.append( \
        str(Path(next(find_default_config_files())).parent.joinpath('pylint/plugins'))
    ) \
    """
load-plugins = [
    "pylint.extensions.code_style",
    "pylint.extensions.typing",
    "hass_constructor",
    "hass_enforce_type_hints",
    "hass_imports",
    "hass_logger",
]
persistent = false
extension-pkg-allow-list = [
    "av.audio.stream",
    "av.stream",
    "ciso8601",
    "orjson",
    "cv2",
]
fail-on = [
    "I",
]
```

Reply:
We could change to this:
```
load-plugins = [
    "pylint.extensions.code_style",
    "pylint.extensions.typing",
]
```
Then we are in line with Core, but it leads to some more work :)
```
... pylint-ing ...
************* Module plugwise.parser
plugwise/parser.py:78:15: R6103: Use 'if (header_index := self._buffer.find(MESSAGE_HEADER)) == -1:' instead (consider-using-assignment-expr)
plugwise/parser.py:93:19: R6103: Use 'if (footer_index := self._buffer.find(MESSAGE_FOOTER)) == -1:' instead (consider-using-assignment-expr)
************* Module plugwise.constants
plugwise/constants.py:432:54: R6101: Consider using namedtuple or dataclass for dictionary values (consider-using-namedtuple-or-dataclass)
plugwise/constants.py:448:56: R6101: Consider using namedtuple or dataclass for dictionary values (consider-using-namedtuple-or-dataclass)
plugwise/constants.py:472:64: R6101: Consider using namedtuple or dataclass for dictionary values (consider-using-namedtuple-or-dataclass)
plugwise/constants.py:530:43: R6101: Consider using namedtuple or dataclass for dictionary values (consider-using-namedtuple-or-dataclass)
************* Module plugwise.smile
plugwise/smile.py:282:15: R6103: Use 'if (power_data := self._power_data_from_location(details['location'])) is not None:' instead (consider-using-assignment-expr)
plugwise/smile.py:405:31: R6102: Consider using an in-place tuple instead of list (consider-using-tuple)
plugwise/smile.py:519:24: R6102: Consider using an in-place tuple instead of list (consider-using-tuple)
plugwise/smile.py:608:11: R6103: Use 'if (subject := contexts.find(locator)) is None:' instead (consider-using-assignment-expr)
plugwise/smile.py:754:15: R6103: Use 'if (lock_state := self._appliances.find(locator).text) == 'true':' instead (consider-using-assignment-expr)
************* Module plugwise.helper
plugwise/helper.py:95:11: R6103: Use 'if (model := version_to_model(name)) != 'Unknown':' instead (consider-using-assignment-expr)
plugwise/helper.py:168:42: R6102: Consider using an in-place tuple instead of list (consider-using-tuple)
plugwise/helper.py:447:15: R6103: Use 'if (module := self._modules.find(locator)) is not None:' instead (consider-using-assignment-expr)
plugwise/helper.py:664:15: R6103: Use 'if (appl := self._appliance_info_finder(appliance, appl)) is None:' instead (consider-using-assignment-expr)
plugwise/helper.py:1272:11: R6103: Use 'if (appl_class := xml.find('type').text) not in SPECIAL_PLUG_TYPES:' instead (consider-using-assignment-expr)
************* Module plugwise.util
plugwise/util.py:69:7: R6103: Use 'if (temp_int := int(seq_id, 16) + value) >= 65532:' instead (consider-using-assignment-expr)
************* Module plugwise.nodes.circle
plugwise/nodes/circle.py:677:15: R6103: Use 'if (_log_timestamp := getattr(message, 'logdate%d' % (_slot, )).value) is None:' instead (consider-using-assignment-expr)
plugwise/nodes/circle.py:827:11: R6103: Use 'if (_slot := self._energy_last_populated_slot) == 0:' instead (consider-using-assignment-expr)
************* Module plugwise.connections
plugwise/connections/__init__.py:56:15: R6103: Use 'if (data := self._read_data()):' instead (consider-using-assignment-expr)
```

Reply:
Lets just make a new issue out of this and merge this one (as in, leave it out for now so it oesn't get bigger)